src/main/java/com/minecolonies/api/entity/mobs/drownedpirate/AbstractDrownedEntityPirate.java

```diff
@@ -0,0 +1,133 @@
+package com.minecolonies.api.entity.mobs.drownedpirate;
+
+import com.minecolonies.api.MinecoloniesAPIProxy;
+import com.minecolonies.api.entity.mobs.AbstractEntityRaiderMob;
+import com.minecolonies.api.entity.mobs.RaiderType;
+import com.minecolonies.api.entity.pathfinding.registry.IPathNavigateRegistry;
+import com.minecolonies.core.entity.pathfinding.navigation.AbstractAdvancedPathNavigate;
+import com.minecolonies.core.entity.pathfinding.navigation.PathingStuckHandler;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.entity.MobSpawnType;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.LevelAccessor;
+import net.minecraft.world.level.LevelReader;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Random;
+
+import static com.minecolonies.api.util.constant.RaiderConstants.ONE;
+import static com.minecolonies.api.util.constant.RaiderConstants.OUT_OF_ONE_HUNDRED;
+
+/**
+ * Abstract for all drowned pirate entities.
+ */
+public abstract class AbstractDrownedEntityPirate extends AbstractEntityRaiderMob
+{
+    /**
+     * Swim speed for pirates
+     */
+    private static final double PIRATE_SWIM_BONUS = 3.0;
+
+    /**
+     * Amount of unique pirate textures.
+     */
+    private static final int PIRATE_TEXTURES = 4;
+
+    /**
+     * Texture id of the pirates.
+     */
+    private int textureId;
+
+    /**
+     * Constructor method for Abstract Barbarians.
+     *
+     * @param type  the type.
+     * @param world the world.
+     */
+    public AbstractDrownedEntityPirate(final EntityType<? extends AbstractDrownedEntityPirate> type, final Level world)
+    {
+        super(type, world);
+        this.textureId = new Random().nextInt(PIRATE_TEXTURES);
+    }
+
+    @Override
+    public void playAmbientSound()
+    {
+        final SoundEvent soundevent = this.getAmbientSound();
+
+        if (soundevent != null && level().random.nextInt(OUT_OF_ONE_HUNDRED) <= ONE)
+        {
+            this.playSound(soundevent, this.getSoundVolume(), this.getVoicePitch());
```

Comment:
do we want to do normal sounds while under water? maybe check head for waterblock and just play drowned zombie noise?